subsys/bluetooth/host/Kconfig

```diff
@@ -155,10 +155,22 @@ config BT_RX_STACK_SIZE
 
 config BT_RX_PRIO
 	# Hidden option for Co-Operative Rx thread priority
+	# HCI drivers that select BT_RECV_IS_RX_THREAD should call bt_recv from
+	# this priority.
+	# Otherwise the host will process events in this priority.
```

Comment:
I would suggest to remove this line. It is a bit confusing

Reply:
I agree towards removal of explanations of hidden Kconfig options as users dont really need to read raw Kconfig files, if required to keep an explanation, it should be in a single place, dummy.c where the checks are placed.